rllib/policy/eager_tf_policy.py

```diff
@@ -823,9 +838,9 @@ def _compute_gradients(self, samples):
                 grads = [g for g, _ in grads_and_vars]
 
             stats = self._stats(self, samples, grads)
-            return grads_and_vars, stats
+            return grads_and_vars, grads, stats
 
-        def _apply_gradients(self, grads_and_vars):
+        def _apply_gradients_helper(self, grads_and_vars):
```

Comment:
same, should we increment retrace_counter here as well?

Reply:
done